Index: tutorial.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n\r\n    <meta charset=\"UTF-8\">\r\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\r\n    <title>Informational Links</title>\r\n    <link rel=\"stylesheet\" href=\"style.css\">\r\n    <header>\r\n        <h1>Frndly QA Knowledge Base</h1>\r\n        <nav>\r\n            <a href=\"index.html\">Home</a>\r\n            <a href=\"about.html\">About</a>\r\n            <a href=\"tutorial.html\">Tutorial</a>\r\n        </nav>\r\n    </header>\r\n<body>\r\n    <h1>Useful Resources</h1>\r\n    <div class=\"links-container\">\r\n        <ul>\r\n            <li><a href=\"https://maven.apache.org/\" target=\"_blank\">Maven</a></li>\r\n            <li><a href=\"https://www.selenium.dev/\" target=\"_blank\">Selenium</a></li>\r\n        </ul>\r\n    </div>\r\n</body>\r\n</html>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tutorial.html b/tutorial.html
--- a/tutorial.html	(revision ce677f23e99a262476c9cec03d5366f7e1c1e031)
+++ b/tutorial.html	(date 1755619840010)
@@ -19,6 +19,7 @@
         <ul>
             <li><a href="https://maven.apache.org/" target="_blank">Maven</a></li>
             <li><a href="https://www.selenium.dev/" target="_blank">Selenium</a></li>
+            <li><a href="https://www.selenium.dev/" target="_blank">Selenium</a></li>
         </ul>
     </div>
 </body>
